ports/openmw-osg/portfile.cmake

```diff
@@ -26,9 +36,20 @@ set(OPTIONS "")
 if(VCPKG_TARGET_IS_MINGW)
     list(APPEND OPTIONS -D_OPENTHREADS_ATOMIC_USE_WIN32_INTERLOCKED=0 -D_OPENTHREADS_ATOMIC_USE_GCC_BUILTINS=1)
 elseif(VCPKG_TARGET_IS_WINDOWS)
-    list(APPEND OPTIONS -D_OPENTHREADS_ATOMIC_USE_WIN32_INTERLOCKED=1 -D_OPENTHREADS_ATOMIC_USE_GCC_BUILTINS=0)
-elseif(VCPKG_TARGET_IS_IOS)
-    # handled by osg
+  list(APPEND OPTIONS
+    -D_OPENTHREADS_ATOMIC_USE_WIN32_INTERLOCKED=1
+    -D_OPENTHREADS_ATOMIC_USE_GCC_BUILTINS=0
+    -DPKG_CONFIG_USE_CMAKE_PREFIX_PATH=ON
+    -DDYNAMIC_OPENSCENEGRAPH=${OSG_DYNAMIC}
+    -DDYNAMIC_OPENTHREADS=${OSG_DYNAMIC}
```

Comment:
Why have these become Windows-only? If MacOS only supports specifically static or specifically dynamic, then dealing with that in a MacOS-specific way (e.g. setting `OSG_DYNAMIC` on MacOS, although that might not be the best way) would be preferable to making it look like Windows is the weird system for supporting the default way of doing things.

Reply:
Parity based on how we are doing it here https://gitlab.com/OpenMW/openmw-dep/-/blob/main/macos/CMakeLists.txt?ref_type=heads

Reply:
I'll try it out and see if it works.